### ISSUE TITLE
Webdev serve: Bad state: Cannot add event after closing

### ISSUE THREAD

**Issue by ruudvring** [dart-lang/webdev#945]
webdev 2.5.6

webdev serve web:8080 --log-requests --no-release

Constant crashes after closing chrome/stopping debugger from vscode:

```
[INFO] Updating asset graph completed, took 5ms
[INFO] Running build completed, took 26.4s
[INFO] Caching finalized dependency graph completed, took 409ms
[INFO] Succeeded after 26.9s with 6 outputs (8 actions)
Unhandled exception:
Bad state: Cannot add event after closing
#0      _StreamController.add (dart:async/stream_controller.dart:587:24)
#1      _StreamSinkWrapper.add (dart:async/stream_controller.dart:860:13)
#2      DevHandler._emitBuildResults (package:dwds/src/handlers/dev_handler.dart:108:31)
#3      _RootZone.runUnaryGuarded (dart:async/zone.dart:1316:10)
#4      _BufferingStreamSubscription._sendData (dart:async/stream_impl.dart:338:11)
#5      _BufferingStreamSubscription._add (dart:async/stream_impl.dart:265:7)
#6      _SyncBroadcastStreamController._sendData (dart:async/broadcast_stream_controller.dart:377:20)
#7      _BroadcastStreamController.add (dart:async/broadcast_stream_controller.dart:252:5)
#8      Stream.asyncMap.onListen.<anonymous closure> (dart:async/stream.dart:522:22)
#9      _RootZone.runUnaryGuarded (dart:async/zone.dart:1316:10)
#10     _BufferingStreamSubscription._sendData (dart:async/stream_impl.dart:338:11)
#11     _DelayedData.perform (dart:async/stream_impl.dart:593:14)
#12     _StreamImplEvents.handleNext (dart:async/stream_impl.dart:709:11)
#13     _PendingEvents.schedule.<anonymous closure> (dart:async/stream_impl.dart:669:7)
#14     _microtaskLoop (dart:async/schedule_microtask.dart:43:21)
#15     _startMicrotaskLoop (dart:async/schedule_microtask.dart:52:5)
#16     _runPendingImmediateCallback (dart:isolate-patch/isolate_patch.dart:118:13)
#17     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:175:5)
```

and also:

```
[INFO] Updating asset graph completed, took 5ms
[INFO] Running build completed, took 26.6s
[INFO] Caching finalized dependency graph completed, took 340ms
[INFO] Succeeded after 26.9s with 6 outputs (8 actions)
[INFO] ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
[INFO] Updating asset graph...2020-04-02T13:55:37.414425  0:00:18.656564 GET     [200] /index.dev.html?debug&react_perf
[INFO] Updating asset graph completed, took 18ms
[INFO] 6.6s elapsed, 3/5 actions completed.ERROR - 2020-04-02 13:56:02.768279
Asynchronous error
Bad state: StreamSink is closed
dart:convert                                    StringConversionSinkMixin.add
package:sse/src/server/sse_handler.dart 73:15   SseConnection._setUpListener
===== asynchronous gap ===========================
package:sse/src/server/sse_handler.dart 53:15   new SseConnection
package:sse/src/server/sse_handler.dart 161:26  SseHandler._createSseConnection.<fn>

[WARNING] No actions completed for 15.1s, waiting on:
  - build_web_compilers:ddc on lib/darwin.ddc.module

[INFO] Running build completed, took 23.5s
[INFO] Caching finalized dependency graph completed, took 652ms
[INFO] Succeeded after 24.2s with 6 outputs (8 actions)
Unhandled exception:
Bad state: Cannot add event after closing
#0      _StreamController.add (dart:async/stream_controller.dart:587:24)
#1      _StreamSinkWrapper.add (dart:async/stream_controller.dart:860:13)
#2      DevHandler._emitBuildResults (package:dwds/src/handlers/dev_handler.dart:108:31)
#3      _RootZone.runUnaryGuarded (dart:async/zone.dart:1316:10)
#4      _BufferingStreamSubscription._sendData (dart:async/stream_impl.dart:338:11)
#5      _BufferingStreamSubscription._add (dart:async/stream_impl.dart:265:7)
#6      _SyncBroadcastStreamController._sendData (dart:async/broadcast_stream_controller.dart:377:20)
#7      _BroadcastStreamController.add (dart:async/broadcast_stream_controller.dart:252:5)
#8      Stream.asyncMap.onListen.<anonymous closure> (dart:async/stream.dart:522:22)
#9      _RootZone.runUnaryGuarded (dart:async/zone.dart:1316:10)
#10     _BufferingStreamSubscription._sendData (dart:async/stream_impl.dart:338:11)
#11     _DelayedData.perform (dart:async/stream_impl.dart:593:14)
#12     _StreamImplEvents.handleNext (dart:async/stream_impl.dart:709:11)
#13     _PendingEvents.schedule.<anonymous closure> (dart:async/stream_impl.dart:669:7)
#14     _microtaskLoop (dart:async/schedule_microtask.dart:43:21)
#15     _startMicrotaskLoop (dart:async/schedule_microtask.dart:52:5)
#16     _runPendingImmediateCallback (dart:isolate-patch/isolate_patch.dart:118:13)
#17     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:175:5)
```

**Comment by grouma** [comment]
Duplicate of https://github.com/dart-lang/webdev/issues/943
